home-assistant-plugin/custom_components/xknx/manifest.json

```diff
@@ -2,6 +2,6 @@
   "domain": "xknx",
   "name": "KNX",
   "documentation": "https://www.home-assistant.io/integrations/knx",
-  "requirements": [],
-  "codeowners": ["@Julius2342"]
+  "requirements": ["xknx==0.12.0"],
```

Comment:
remove the requirement for the custom component so it always uses the local checked out version

Reply:
I added a new version of the scripts that do this stuff automatically.

For backport: simply remove it
For upstream PR: get latest release version from github and use that as requirement